src/negbinfit.jl

```diff
@@ -34,6 +34,20 @@ function mle_for_θ(y::AbstractVector, μ::AbstractVector, wts::AbstractVector;
     θ
 end
 
+"""
+    negbin(formula, data, link;
+           initialθ::Real=Inf, maxIter::Integer=30, convTol::Real=1.e-6,
+           verbose::Bool=false, kwargs...)
+
+Fit a negative binomial generalized linear model to data, while simultaneously
+estimating the shape parameter θ. `maxIter`, `convTol`, and `verbose` correspond
+to the keyword arguments of the same name used by [`glm`](@ref). Extra arguments
+and keyword arguments will be passed to [`glm`](@ref).
+
+# Keyword Arguments
+- `initialθ::Real=Inf`: Starting values for shape parameter θ. If it is `Inf`
```

Comment:
Singular "value".

Also, better document other arguments here rather than above, that's more logical.

Reply:
done